tests/Unit/InvalidLambda/empty_lambda2.cpp

```diff
@@ -5,6 +5,6 @@ int main()
     int gpu_result;
 	concurrency::array_view<int> gpu_resultsv(1, &gpu_result);
     gpu_resultsv.discard_data();
-    auto fun = [&]() restrict(cpu,amp) { return 0; };
+    static auto fun = [&]() restrict(cpu,amp) { return 0; };

```

Comment:
@AlexVlx could you elaborate why `static` , `inline` are needed now in these tests? I assume something in `CodeGenModule` has changed the ordering for lambdas?